src/routes/users.js

```diff
@@ -12,10 +12,13 @@ const routes = express.Router();
 
 routes.post('/signup', signupVatidate, checkUserExist, Users.signup);
 routes.get('/verify-email/:token', Users.verifyEmail);
+routes.patch('/change-password', Users.changePassword);
```

Comment:
protect this route
make it like 
`routes.patch('/change-password', isAuthenticated, Users.changePassword);`

Reply:
yes i am doing it the issues i am having problem with mocha